server/app/auth/oidc/admin/AdfsProfileCreator.java

```diff
@@ -23,14 +20,11 @@ public class AdfsProfileCreator extends CiviformOidcProfileCreator {
   private final String ad_groups_attribute_name;
 
   public AdfsProfileCreator(
-      OidcConfiguration configuration,
-      OidcClient client,
-      ProfileFactory profileFactory,
-      Config appConfig,
-      Provider<AccountRepository> accountRepositoryProvider) {
-    super(configuration, client, profileFactory, accountRepositoryProvider);
-    this.adminGroupName = appConfig.getString("adfs.admin_group");
-    this.ad_groups_attribute_name = appConfig.getString("adfs.ad_groups_attribute_name");
+      OidcConfiguration oidcConfiguration, OidcClient client, OidcClientProviderParams params) {
+    super(oidcConfiguration, client, params);
+    this.adminGroupName = params.configuration().getString("adfs.admin_group");
+    this.ad_groups_attribute_name =
+        params.configuration().getString("adfs.ad_groups_attribute_name");
```

Comment:
**nitpick (if-minor):** since you're touching this line would you please rename the variable to be camelCase?

Reply:
Done